src/components/SearchTracePage/index.js

```diff
@@ -18,29 +18,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import _values from 'lodash/values';
```

Comment:
Is this a common way of naming single lodash functions? Historically, this is generally meant for private variables.

Are we opposed to bring in the whole lib because of size? If so, I suppose we could do this style as long as we are consistent throughout the application.